api/bases/cinder.openstack.org_cinderapis.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.1
+    controller-gen.kubebuilder.io/version: v0.10.0
```

Comment:
you probably have an older version of `controller-gen` within your repo because this shouldn't change. 
(the same comment applies to the other files).

Reply:
whoa! I knew I was working on an older system that has an old controller-gen, but accidentally "git add -u" when I only meant to update a single file.